build/AzurePipelinesTemplates/MUX-InstallWindowsSDK-Steps.yml

```diff
@@ -8,3 +8,30 @@ steps:
       filePath: build\Install-WindowsSdkISO.ps1
       arguments: ${{ parameters.sdkVersion }}
     displayName: 'Install Windows SDK (${{ parameters.sdkVersion }})'
+
+  - task: 333b11bd-d341-40d9-afcf-b32d5ce6f23b@2
+    condition: ne(variables['System.CollectionUri'],'https://dev.azure.com/ms/')
```

Comment:
What I would do is update the Internal Pipeline to add a Variable named "UseInternalSDK", default to 'true' and settable at queue time.

Then update these conditions to use that variable. Remove the 'useInsiderSDK' parameter (different from a variable), and use the variable for VSBuild task.

Let me know if you need any help with that.

Reply:
Done...and running the 'Internal pipeline', it is indeed using 10.0.21307.0 as expected.